fog/overseer/server/src/service.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use crate::{error::OverseerError, worker::OverseerWorker};
+use mc_common::logger::{log, Logger};
+use mc_fog_recovery_db_iface::RecoveryDb;
+use mc_fog_uri::FogIngestUri;
+use std::sync::{
+    atomic::{AtomicBool, Ordering},
+    Arc,
+};
+
+// Implements core logic for the Fog Overseer HTTP server.
+pub struct OverseerService<DB: RecoveryDb + Clone + Send + Sync + 'static>
+where
+    OverseerError: From<DB::Error>,
+{
+    ingest_cluster_uris: Vec<FogIngestUri>,
+    logger: Logger,
+    overseer_worker: Option<OverseerWorker>,
+    recovery_db: DB,
+    is_enabled: Arc<AtomicBool>,
+    stop_requested: Arc<AtomicBool>,
```

Comment:
No need to store this in this struct since it doesn't look like it's ever being accessed.

Reply:
Nevermind, we can remove it.